pkg/reconciler/revision/table_test.go

```diff
@@ -155,18 +169,24 @@ func TestReconcile(t *testing.T) {
 			InduceFailure("create", "deployments"),
 		},
 		Objects: []runtime.Object{
-			rev("foo", "create-user-deploy-failure"),
+			rev("foo", "create-user-deploy-failure", WithContainerStatuses([]v1.ContainerStatuses{
+				{Name: "create-user-deploy-failure", ImageDigest: "busybox@sha256:deadbeef"},
+			})),
```

Comment:
This gets pretty repetitive still - what if we have a have a helper that generates the stub's values
```suggestion
			rev("foo", "create-user-deploy-failure", WithDefaultContainerStatuses()),
```

```go
func WithDefaultContainerStatuses() RevisionOption {
  return func(r *v1.Revision) {
    r.Status.ContainerStatuses = []v1.ContainerStatuses{
      ... use `r` to set the name  etc ...
    }
  }
}
```